packages/cf-typography/usage.md

```diff
@@ -37,8 +37,50 @@ and has more basic typography patterns.
 
 ## Variables
 
-Theme variables for setting the color and sizes. Overwrite them in your own
-project by duplicating the variable `@key: value`.
+Component variables are used to theme a component.
+They likely will be left as is, but if needed can be overwritten by duplicating
+the variable in a `@key: value` format with a different value.
+This customized variable would be placed in the same file
+where this component's less file is imported.
+
+### Color variables
+
+Color variables referenced in comments are from [cf-core cf-brand-colors.less](https://github.com/cfpb/capital-framework/blob/master/packages/cf-core/src/cf-brand-colors.less).
+
+```
+// Running text elements
+
+// .a-micro-copy
+@micro-copy:                @black;
+
+// .a-date
+@date:                      @gray;
+
+// .m-pull-quote
+@pull-quote_body:           @black;
+@pull-quote_citation:       @gray;
+
+// Headings
+
+// .a-heading__icon
+@heading__icon:             @black;
+@heading__icon__hover:      @link-text-hover;
+
+// Headers
+
+// .m-slug-header
+@slug-header_border__thin:  @gray-10;
+@slug-header_border__thick: @green;
+
+// .m-meta-header
+@meta-header_border:        @gray-40;
+
+// Links
+
+// .a-link__jump
+@jump-link_bg:              @gray-10;
+@jump-link_border:          @gray-40;
+```
 
 ### Font variables
```

Comment:
The TOC entry for this needs to be updated. Apparently, it used to be "Font source variables".

Reply:
It has both "Font variables" and "Font source variables"
I added the missing TOC entry, but we may want to combine or nest those.